oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -35,11 +35,11 @@ Context
 
 Developers who work on the Open edX platform, both inside and outside of edX, need accurate and current documentation about platform architecture, APIs, and development best practices.
 
-**Developer documentation** is defined as written specification, instructions, and resources for software developers who extend and contribute to the Open edX platform. It does not include documentation resources targeted for edX learners, educators, researchers, and Open edX operators. Those are out of scope of this OEP and maintained by the edX documentation team at docs.edx.org_.
+**Developer documentation** is defined as written specification, instructions, and resources for software developers who extend and contribute to the Open edX platform. It does not include documentation resources targeted for edX learners, educators, researchers, and Open edX operators. Those are out of scope of this OEP and maintained by the edX at docs.edx.org_.
```

Comment:
`the edX` => `edX` or `edX.org`? Also, since Open edX ownership post 2U is still in question, maybe we should:
* Not state anything about the owner, and just point to `docs.edx.org`, or
* Point to any process docs for updating `docs.edx.org`, or
* Something of this nature?